src/pe/mod.rs

```diff
@@ -216,6 +220,29 @@ impl<'a> PE<'a> {
                 )?);
             }
 
+            if let Some(&tls_table) = optional_header.data_directories.get_tls_table() {
+                tls_data = if is_64 {
+                    tls::TlsData::parse_with_opts::<u64>(
+                        bytes,
+                        image_base,
+                        &tls_table,
```

Comment:
nit: shouldn't need `&` here and above in `let Some(&tls_table)`

Reply:
Done.